src/include/migraphx/context.hpp

```diff
@@ -373,7 +373,7 @@ struct context
     private_detail_te_handle_base_type& private_detail_te_get_handle()
     {
         assert(private_detail_te_handle_mem_var != nullptr);
-        if(not private_detail_te_handle_mem_var.unique())
+        if(private_detail_te_handle_mem_var.use_count() != 1)
```

Comment:
These changes will be overwritten when calling `make generate`.

Reply:
The problem is `unique()` is deprecated on Windows and fails to compile due to the C++17 standard used. So what can I do to make it persistent?

Reply:
Sure, it looks like you put the change into te.py which is the correct place for it. If not thats where the change would need to be.